lib/models/github-user.js

```diff
@@ -4,6 +4,10 @@ const logger = require('../logger');
 const Sequelize = require('sequelize');
 const EncryptedField = require('sequelize-encrypted');
 
+if (!process.env.STORAGE_SECRET) {
+  throw 'STORAGE_SECRET is not defined.'
```

Comment:
`throw new Error('STORAGE_SECRET is not defined.');` should fix both of these lint failures:

```
  8:3   error  Expected an object to be thrown  no-throw-literal
  8:41  error  Missing semicolon                semi
```